wpeverest/user-registration
# Feature - Introduction to {{passwordless_login_link}} Smart Tag.

### All Submissions:



### Changes proposed in this Pull Request:

Previously, there was no template for modifying email of 'Passwordless Login Email' from Settings > Email. Further, there was no feature to override 'Passwordless Login Email' from Form Settings > Email Templates. This PR fixes this. (Activate Email Templates)
 
UR-Pro Dependency: https://github.com/wpeverest/user-registration-pro/pull/324
UR-Email Templates Dependency: https://github.com/wpeverest/user-registration-email-templates/pull/20

Closes # .

### How to test the changes in this Pull Request:

1. Enable Passwordless Login option from General Settings > Login Options.
2. Test how Passwodless login was working before and what email was being sent.
3. Switch to this branch, Goto Settings > Emails > Passwordless Login. 
4. Modify the content and verify this by logging in using Passwordless Login.
5. Goto Form Settings > Email Templates, Override the Passwordless Login email Content.
6. Again verify by performing Passwordless Login.

### Types of changes:



### Other information:



### Changelog entry

> Feature - Introduction to {{passwordless_login_link}} Smart Tag.
